RangeBarSample/src/main/res/values/strings.xml

```diff
@@ -38,6 +38,7 @@
     <string name="check_box_rounded_background">Rounded background</string>
     <string name="top_tick_labels">Top tick labels</string>
     <string name="bottom_tick_labels">Bottom tick labels</string>
+    <string name="recyclerview_text">Check with RecyclerView</string>
```

Comment:
This could be renamed to `Rangebar Recyclerview Implementation"

Reply:
Done.